scalding-core/src/main/scala/com/twitter/scalding/CumulativeSum.scala

```diff
@@ -39,9 +39,9 @@ object CumulativeSum {
     def cumulativeSum(
       implicit sg: Semigroup[V], ordU: Ordering[U], ordK: Ordering[K]): SortedGrouped[K, (U, V)] = {
       pipe.group
-        .sortBy { case (u: U, _) => u }
+        .sortBy { case (u, _) => u }
```

Comment:
@isnotinvain Dropping the types since there're warnings about type erasure here. 



[warn] /Users/exie/learn/ScaldingGithub/scalding/scalding-core/src/main/scala/com/twitter/scalding/CumulativeSum.scala:42: abstract type pattern U is unchecked since it is eliminated by erasure
[warn]         .sortBy { case (u: U, _) => u }
[warn]                            ^
[warn] /Users/exie/learn/ScaldingGithub/scalding/scalding-core/src/main/scala/com/twitter/scalding/CumulativeSum.scala:44: abstract type pattern U is unchecked since it is eliminated by erasure
[warn]           case (acc, (u: U, v: V)) =>
[warn]                          ^
[warn] /Users/exie/learn/ScaldingGithub/scalding/scalding-core/src/main/scala/com/twitter/scalding/CumulativeSum.scala:44: abstract type pattern V is unchecked since it is eliminated by erasure
[warn]           case (acc, (u: U, v: V)) =>
[warn]                                ^
[warn] /Users/exie/learn/ScaldingGithub/scalding/scalding-core/src/main/scala/com/twitter/scalding/CumulativeSum.scala:66: abstract type pattern U is unchecked since it is eliminated by erasure
[warn]         .map { case (k, (u: U, v: V)) => (k, partition(u)) -> v }
[warn]                             ^
[warn] /Users/exie/learn/ScaldingGithub/scalding/scalding-core/src/main/scala/com/twitter/scalding/CumulativeSum.scala:66: abstract type pattern V is unchecked since it is eliminated by erasure
[warn]         .map { case (k, (u: U, v: V)) => (k, partition(u)) -> v }
[warn]                                   ^
[warn] /Users/exie/learn/ScaldingGithub/scalding/scalding-core/src/main/scala/com/twitter/scalding/CumulativeSum.scala:90: abstract type pattern U is unchecked since it is eliminated by erasure
[warn]           case (k, (u: U, v: V)) =>
[warn]                        ^
[warn] /Users/exie/learn/ScaldingGithub/scalding/scalding-core/src/main/scala/com/twitter/scalding/CumulativeSum.scala:90: abstract type pattern V is unchecked since it is eliminated by erasure
[warn]           case (k, (u: U, v: V)) =>